quaternionmedia/codecartographer
# 22 Handle env files

Passing env into docker secrets so you don't have to manually create each time.
Updated a few of the error handling aspects as well for cleaner logs.

setup:
1. Create a token.txt with github api token inside
2. build docker 
3. open site

test steps:
1. click parser
2. input a valid public repo 
3. click submit
     - Will see results without having to have manually created .env
4. input an invalid url 
5. click submit
     - Will see a message saying url is invalid 
6. input valid url again
7. click submit
8. open contents and find a non .py file 
9. click 'plot' 
     - will see plot page with buttons disabled and message saying invalid file type
10. return to parse page 
11. click 'plot' on a .py file
      - will see plot page with buttons enabled 
12. click 'single'
      - will see the plot 
13. open docker desktop
14. view the logs for container
      - will see a cleaner log of errors
15. on site click 'return to home'
16. click palette
      - will see the current palette
17. return home and click plot
18. click 'single'
      - will see the plot show

Closes #22
